pulp_ansible/app/viewsets.py

```diff
@@ -113,7 +111,7 @@ class CollectionVersionFilter(ContentFilter):
 
     namespace = filters.CharFilter(field_name="namespace")
     name = filters.CharFilter(field_name="name")
-    is_highest = filters.BooleanFilter(field_name="is_highest", method="get_highest")
+    is_highest = filters.BooleanFilter(field_name="is_highest")
```

Comment:
Is it correct to say that we don't need this because the model contains it in the DB: https://github.com/pulp/pulp_ansible/blob/f4164bd007adc19e2dae5edb28d0dfc5ade885ce/pulp_ansible/app/models.py#L144

Reply:
yes, we did a data migration in the past to assure that:
https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/migrations/0005_collectionversion_is_highest.py